feature/office/src/main/java/com/iteneum/office/data/OfficeModel.kt

```diff
@@ -14,6 +11,6 @@ import com.itenum.domain.Email
 data class OfficeModel(
```

Comment:
Rename OfficeModel to OfficeDTO 

Note: when a model is in data layer is named as DTO, when is in domain Layer it should named as Office or OfficeDomain

Reply:
already renamed to Office and moved into domain
![imagen](https://github.com/irv205/droid_leaspert/assets/128720218/f019f30f-766d-454a-aaf6-1ee082104655)